### PR TITLE
`Development`: Fix multi node docker setup for macOS and update docker images

### PR DESCRIPTION
**Pull request by krusche**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Keep things up to date

### Steps for Testing

- Test servers are working
- Docker configs run locally independent of OS

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### CODE REVIEW

**Review by az108 — APPROVED** [review]
Works as expected, tested in testing session together and tested it myself aswell

**Review by SimonEntholzer — DISMISSED** [review]
Works as expected during testing.

**Review by florian-glombik — CHANGES_REQUESTED** [review]
I think that I found a mistake in `docker/test-server-multi-node-postgresql-localci.yml`

Thanks for taking the time to introduce anchors and thereby improving future maintenance and adjustments 🙏

File: docker/test-server-multi-node-postgresql-localci.yml [hunk 1]
```diff
@@ -25,67 +27,37 @@ services:
...
-            - ${DOCKER_GROUP_ID:-0}
         env_file:
             - ${ARTEMIS_ENV_FILE:-./artemis/config/prod-multinode.env}
-            - ${ARTEMIS_NODE_3_ENV_FILE:-./artemis/config/node3.env}
-        volumes:
-            - ${ARTEMIS_VOLUME_MOUNT:-./.docker-data/artemis-data}:/opt/artemis/data
-            - ${ARTEMIS_LEGAL_MOUNT:-./.docker-data/artemis-legal}:/opt/artemis/legal
-            - ${ARTEMIS_DATA_EXPORT_MOUNT:-./.docker-data/artemis-data-exports}:/opt/artemis/data-exports
-            - /var/run/docker.sock:/var/run/docker.sock
+            - ${ARTEMIS_NODE_2_ENV_FILE:-./artemis/config/node3.env}
```
[the remark this answers is not in the document]
Comment (BBesrour) [reply docker/test-server-multi-node-postgresql-localci.yml]: You are right, nice catch!

**Review by az108 — APPROVED** [review]
reapprove

**Review by florian-glombik — APPROVED** [review]
The code looks good to me, thanks for introducing yaml anchors!

**Review by marlon-luca-bu — APPROVED** [review]
Code

**Review by sarpsahinalp — COMMENTED** [review]
Multi node mysql setup works without errors, but the multi node postgres setup gives me the following error:
image
I am using a Linux setup

**Review by sarpsahinalp — APPROVED** [review]
Works after giving permissions to postgres folders

**Pull request merged**